vuepress/docs/next/tutorials/create/mfe/react.md

```diff
@@ -13,23 +13,25 @@
    ent bundle init simple-bundle
    ```
 
-2. Add an MFE to the bundle project structure:
+2. From the root bundle folder, add an MFE to the bundle project structure:
```

Comment:
adding an MFE is not really adding it to the structure, so this seems misleading, like it's only a structural thing when it adds a folder but also adds it to the entando.json

Reply:
disagree, because structure pertains to both file contents and directory, like how code is written as well as file hierarchy. i'll remove "structure" to keep both implied without an extra word